public/index.html

```diff
@@ -6,7 +6,7 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no"> 
     <link rel="stylesheet" type="text/css" href="index.css">
-    <script src="./index.js" type="module" defer></script>
```

Comment:
I'm leaving module be, because I like it, but by default it says to defer, so we don't need it. But should we change this? modules have different global behaviors that require imports, do the students know these? We may actually want to swap module for defer for now. Your call!